tests/system/src/test/java/test/robot/testharness/VisualTestBase.java

```diff
@@ -106,7 +107,7 @@ public static void doTeardownOnce() {
 
     @Before
     public void doSetup() {
-        runAndWait(() -> robot = com.sun.glass.ui.Application.GetApplication().createRobot());
+        runAndWait(() -> new Robot());
```

Comment:
you lost the "robot =" ... that should be:

`runAndWait(() -> robot = new Robot());`